tfx_bsl/beam/run_inference_test.py

```diff
@@ -568,7 +569,8 @@ def test_request_body_with_binary_data(self):
                 'b64': 'QVNhOGFzZGY='
             },
             'x': 'JLK7ljk3',
-            'y': [1, 2]
+            'y': [1, 2],
+            'z': [4.5, 5, 5.5]
```

Comment:
This test seems not reflect the reason for this change. Could you add a more representative test?

Reply:
I added an additional unit test that tests the full TFExample -> HttpRequest path (test_can_format_requests). Before this PR, this test will fail with the Exception: "RepeatedScalarContainer is not JSON-serializable".